Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -226,12 +226,14 @@ extension BrowserViewController: WKNavigationDelegate {
             return
         }
 
-        UIApplication.shared.open(url, options: [:]) { openedURL in
-            // Do not show error message for JS navigated links or redirect as it's not the result of a user action.
-            if !openedURL, navigationAction.navigationType == .linkActivated {
-                let alert = UIAlertController(title: Strings.UnableToOpenURLErrorTitle, message: Strings.UnableToOpenURLError, preferredStyle: .alert)
-                alert.addAction(UIAlertAction(title: Strings.OKString, style: .default, handler: nil))
-                self.present(alert, animated: true, completion: nil)
+        if !(url.scheme?.contains("firefox") ?? true) {
```

Comment:
I think I should block only "firefox://open?" urls. We also have "firefox://fxa?" urls and "firefox://deep-link?" urls. These might end up in emails that the user views in-browser.

Reply:
I don't think these would or should end up in emails.

Reply:
Cool, will leave as-is, more restrictive is good in this case